pkg/extensions/errorhandler.go

```diff
@@ -0,0 +1,7 @@
+package extensions
+
+import "context"
+
+// ErrorHandler is the signature of the function that needs to be implemented to use
+// errorhandler functionality
+type ErrorHandler func(ctx context.Context, topic string, msg *AcknowledgeableBrokerMessage, err error)
```

Comment:
Linked to the default handler comments.

```suggestion
type ErrorHandler func(ctx context.Context, topic string, msg *AcknowledgeableBrokerMessage, err error)

// DefaultErrorHandler returns the default error handler, which is Noop one.
func DefaultErrorHandler() ErrorHandler {
	return errorhandlers.Noop()
}
```